### PR TITLE
Fix a couple of typos in Events.md

### PR DESCRIPTION
**Pull request by Mokou**

### CODE REVIEW

File: docs/static/Events.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ are all run under the same supervisor. As part of the setup for each shard, it
 creates its own producer under a supervisor.
 
 As events are sent to the shard, the following happens:
- 1. Shard looks to see what type of event it is, only dispatch events are send to
+ 1. Shard looks to see what type of event it is, only dispatch events are sent to
  the producer.
 
  2. If the event is a `Dispatch`, the payload is converted to a an atom-keyed map.
```
Comment (Kraigie) [inline docs/static/Events.md]: Could you also correct the typo on this line? `to a an atom-keyed` -> `to an atom-keyed`

**Comment by Kraigie** [comment]
Thanks!

File: docs/static/Events.md (hunk 1 above)
Comment (Mokou) [inline docs/static/Events.md]: Np, didn't notice this one. Btw I think github gives you write access to push request branches by default.

Comment (Kraigie) [inline docs/static/Events.md]: If I'm understanding you correctly, you're right. I took the easy way out and delegated the work to you. :)

**Pull request merged**